src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.Baselines.cs

```diff
@@ -141,7 +142,7 @@ public static void Main()
 
                             public class MyClass
                             {
-                                public string MyString { get; set; }
+                                public string? MyString { get; set; }
```

Comment:
Testing removal of `EraseCompileTimeMetadata` - https://github.com/dotnet/runtime/pull/89587#discussion_r1334692857.